modules/distr-tracing-install-jaeger-operator.adoc

```diff
@@ -5,40 +5,35 @@ This module included in the following assemblies:
 
 :_mod-docs-content-type: PROCEDURE
 [id="distr-tracing-jaeger-operator-install_{context}"]
-= Installing the {JaegerName} Operator
+= Installing the {JaegerOperator} Operator
 
-To install {JaegerName}, you use the link:https://operatorhub.io/[OperatorHub] to install the {JaegerName} Operator.
+You can install the {JaegerOperator} Operator through the link:https://operatorhub.io/[*OperatorHub*].
 
 By default, the Operator is installed in the `openshift-operators` project.
 
 .Prerequisites
 * You have access to the {product-title} web console.
 * You have access to the cluster as a user with the `cluster-admin` role. If you use {product-dedicated}, you must have an account with the `dedicated-admin` role.
-* If you require persistent storage, you must also install the OpenShift Elasticsearch Operator before installing the {JaegerName} Operator.
-
-[WARNING]
-====
-Do not install Community versions of the Operators. Community Operators are not supported.
-====
+* If you require persistent storage, you must also install the OpenShift Elasticsearch Operator before installing the {JaegerOperator} Operator.
```

Comment:
I suggest using the existing attribute, {es-op}, for the OpenShift Elasticsearch Operator. This is especially important because there are licensing debates around Elasticsearch so who knows what decisions might be taken about it in the future?

I am also not sure the word "also" is needed here.

Reply:
Thank you. Good to know. I've applied your suggestion here manually.